R/repository.R

```diff
@@ -1,18 +1,23 @@
 #' Create or open a git repository
 #'
-#' Use [git_init()] to start a new repository or [git_clone()] to download a
+#' @description
```

Comment:
Keeping the second paragraph up here in description vs in details.

Don't use the link syntax when talking about something documented here in this topic, except in a context where it gets inherited elsewhere and the link is meaningful.